draft-ietf-rtcweb-jsep.xml

```diff
@@ -3540,12 +3546,19 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
               <t>tls-id value, which MUST be set according to
               <xref target="I-D.ietf-mmusic-dtls-sdp" />, Section 5. If
-              this is a re-offer and the tls-id value is different
+              this is a re-offer or a response to a re-offer
+              and the tls-id value is different
               from that presently in use, the DTLS connection is not
               being continued and the remote description MUST be part
               of an ICE restart, together with new ufrag and password
-              values. If this is an answer, the tls-id value, if
-              present, MUST be the same as in the offer.</t>
+              values. Note that although JSEP requires that answers
+              change the tls-id value if and only if the offerer does,
+              answerer, non-JSEP answerers are permitted to change the
+              tls-id as long as the offer contained an ICE restart.
+              Thus, JSEP implementations which process DTLS data prior
```

Comment:
This applies to fingerprint changes as well; suggest moving this to 5.11, where we talk about the need to start a new DTLS connection if either the tls-id or fingerprint changes.